src/vm/comdelegate.cpp

```diff
@@ -2593,15 +2593,6 @@ BOOL COMDelegate::NeedsWrapperDelegate(MethodDesc* pTargetMD)
 {
     LIMITED_METHOD_CONTRACT;
 
-#ifdef _TARGET_ARM_
```

Comment:
Could you remove my email address from DW? I think this discussion does not
concern me.
Cheers
Julian

28 lip 2016 4:06 PM "Jan Kotas" <notifications@github.com> napisał(a):

> In src/vm/comdelegate.cpp
> <https://github.com/dotnet/coreclr/pull/6500#discussion_r72628027>:
>
> > @@ -2593,15 +2593,6 @@ BOOL COMDelegate::NeedsWrapperDelegate(MethodDesc* pTargetMD)
> >  {
> >      LIMITED_METHOD_CONTRACT;
> >
> > -#ifdef _TARGET_ARM_
>
> I believe that implementing COMDelegate::GetSecureInvoke is easier. It
> should be implemented similar to how COMDelegate::GetMulticastInvoke is
> implemented, except that there will be no loop. It should just re-push the
> arguments and call the actual target. The call to actual target generated
> by the JIT has to pass the indirection cell in R4. There will need be a new
> flag passed to the JIT to make it happen.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/coreclr/pull/6500/files/c7d62d06126218318efee2d7bf5ed0158ffb58d5#r72628027>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAA1nOgyJ-Ik3RQUtmQvlY0BKE3L3gPYks5qaLeBgaJpZM4JW0HY>
> .
>